### ISSUE TITLE
Option for exchange dot and comma

### ISSUE THREAD

**Issue by christiandull** [burton999dev/CalcNoteHelp#20]
In europe it is usual to use the comma as decimal seperator.

I tryed to setup it with grammer, but don't get the result i want.

It would be great to have a option to exchange the dot and the comma (in the view and in the keyboard). It should be a simple checkbox or dropdownlist with 2 options (dot as seperator or comma).

**Comment by burton999dev** [comment]
> I tryed to setup it with grammer, but don't get the result i want.

Can you give share with me what you have set up?

**Comment by burton999dev** [comment]
1. [Settings] -> [Calculation Settings] -> [Customize the grammar]
2. Change `Decimal point` as `Comma`
3. Change `Thousands separator` as `Period`
4. Change `Argument separator` as `Period` or others you want. (I'm not sure what the European people use for argument separators.)
5. Tap the save button(Floppy disk icon)

**Comment by christiandull** [comment]
With these settings, it shows at the left side the comma correct and also the comma and dot is switched in the keybaord. But in the result it shows the comma where the dot should be and the dot where the comma should be.

See screenshot
![Screenshot_20211215-142119](https://user-images.githubusercontent.com/51074719/146194625-39f6e705-181e-4d6f-997b-76b114567de6.jpg)
.

**Comment by burton999dev** [comment]
Also you need to change `Result display format`

**Comment by christiandull** [comment]
Oh, i have overseen that.

Now it works perfect how i wish it. 

Maybe it would be a good idea to add a checkbox like i proposed that changes these 4 settings with one click. I think to find out and change the 4 settings is for the most users a little bit complex.

Another question: Is it possible to show the dot also when typing at the left side:

100
then add a additional zero also at the left side and then it should show this
1.000


![Screenshot_20211215-145821](https://user-images.githubusercontent.com/51074719/146200829-5f2044cc-da49-4f1c-b229-f7e1ec1f0f09.jpg)

**Comment by burton999dev** [comment]
> I think to find out and change the 4 settings is for the most users a little bit complex.

I agree with you.
But, countries in the world use single quotation marks as thousand separators, and others use spaces.
Perhaps there are more variations that I just don't know about.
Also, I don't even know what characters the countries that use commas for decimal points use as argument separators.
Perhaps there are several patterns to this as well.
Because of this, I can't keep track of everything, so I decided to leave everything to the users.

> Is it possible to show the dot also when typing at the left side

Unfortunately, CalcNote does not support auto-formatting a thousand separators.
It is very difficult for an app to change the input while the user is typing.
I tried to implement this before, but the movement was unnatural and uncomfortable.
